### ISSUE TITLE
[Question] Talks archive

### ISSUE THREAD

**Issue by nikolaposa** [opencfp/opencfp#622]
I was wondering whether there is an option to archive/reset talks from a previous year before opening CFP for the new year so that they are not visible anymore? Is there a more elegant/meaningful way other than wiping out talks data?

Or to formulate it differently: what's the recommended workflow for reusing CFP between conferences?

**Comment by chartjes** [comment]
@nikolaposa I think the usual process is to simply delete everything from the talks table but keep the user accounts in place.

If I were to implement this in code I would add a column to the talks table that indicates a talk is archived or not and then allow the archiving to occur via the bin/opencfp CLI tool.

**Comment by nikolaposa** [comment]
> If I were to implement this in code I would add a column to the talks table that indicates a talk is archived or not and then allow the archiving to occur via the bin/opencfp CLI tool.
@chartjes That could be a useful feature.

Feel free to close this issue, I got the input I needed. Thanks!

**Comment by localheinz** [comment]
I am not a conference organizer, but I think this could be a good feature in the context of evaluating submissions, for example

* do we want to hear this speaker again talking about a very similar topic
* statistics

**Comment by BackEndTea** [comment]
Europeans may want to worry about the [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) though.

**Comment by mdwheele** [comment]
> Europeans may want to worry about the GDPR though.

And to my understanding, "flagging as archived" would not be compliant. I think it'd require a full deletion of all requested user data. That wouldn't be that bad honestly, especially now that we've shored up better interfaces for `AccountManagement`. 

I recently had to touch GPDR with an Event Store implementation I maintain. Fun stuff. Reasonably easy to comply with (on the surface, at least).
